.pre-commit-config.yaml

```diff
@@ -26,6 +26,7 @@ repos:
     hooks:
     - id: trailing-whitespace
     - id: end-of-file-fixer
+      exclude: interactive/templates/components/link.html
```

Comment:
Why does that need to be in here?

Reply:
Nevermind, I see it now!

Reply:
Ah this is because the link component is inline and the extra space will cause links to have a space after them – and will look like this when used inline:  
`<a href="#">here</a> ).`

Image example:

> <img width="654" alt="CleanShot 2022-05-20 at 17 47 28@2x" src="https://user-images.githubusercontent.com/24863179/169574756-5c7058a4-f90d-4d99-b378-4aab8aa03228.png">


...which is very annoying. Therefore the file needs to have no trailing newline, which means the hook needs to ignore the file.

Reply:
I added a comment to the [component file](https://github.com/opensafely-core/interactive.opensafely.org/blob/eb6841a403e5b39c8b00a33ff3e2a818f5c6096d/interactive/templates/components/link.html#L9) to say this, but I can add one to the hooks file as well if you want?

Reply:
I'd noticed the extra space earlier, and it was indeed, very annoying. I'm glad you've fixed it. 

I think it's fine as-is. When I saw it, I went to the file in question and got my answer.